src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -157,12 +162,17 @@ object DemuxFastqs {
         .flatMap { batch =>
           batch
             .parWith(pool=pool)
-            .map { readRecords => demultiplexer.demultiplex(readRecords: _*) }
+            .map { readRecords => 
+                demultiplexer.demultiplex(readRecords: _*)
+                    .maskLowQualityBases(threshold=thresh, qualityEncoding=qualityEncoding) 
```

Comment:
why is this better than having two `maps`?  I think it is

Reply:
To avoid iterating through all the records again, and instead doing both steps at once?

Reply:
@kstromhaug I believe this will create two "parallel" collections.  The order in which the `map` are applied is non-deterministic, so we could end up having `demultiplex` get called before any `maskLowQualityBases` gets applied, so it could cause a lot more memory usage.